deployment/data-streams/changeset/deploy_configurator_test.go

```diff
@@ -0,0 +1,30 @@
+package changeset
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/testutil"
+)
+
+func TestDeployConfigurator(t *testing.T) {
+	e := testutil.NewMemoryEnv(t, true)
+	cc := DeployConfiguratorConfig{
+		ChainsToDeploy: []uint64{testutil.TestChain.Selector},
+	}
+	out, err := DeployConfigurator{}.Apply(e, cc)
```

Comment:
What do we think about using [this](https://github.com/smartcontractkit/chainlink/blob/9a3f44a7da8c8aedcb9e6fc6444e7251e314cc04/deployment/common/changeset/test_helpers.go#L50-L50) test helper across our tests? Seems like it will handle some boiler plate